source/manual/howto-change-slug-and-create-redirect.html.md

```diff
@@ -25,3 +25,13 @@ $ bundle exec 'reslug:document[old_slug,new_slug]'
 
 > This rake task also works for Person, Role, PolicyGroup, Organisation, WorldOrganisation and WorldLocation.
 For example: `reslug:world_location[old_slug,new_slug]`
+
+#### Issues
+
+If you run the task and find the redirect has worked but the new location returns a 404, it's likely because there's a queue buildup in publishing-api ([check in Grafana](https://grafana.publishing.service.gov.uk/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=publishing-api&var-Queues=All&from=now-30m&to=now).
```

Comment:
I guess I'm still not confident enough about the low/high queue priority being an issue here. I'd prefer to investigate further get more confidence before writing it down (as there's a tendency to believe the stuff in the docs to be true).

Reply:
I've seen this problem before.

The problem lies in: 

- Whitehall publishes to Publishing API as a republish which is low priority
- Publishing API runs a nested command to create the redirect
- nested command (creating redirect) is not a redirect and has high priority and is published first
- the previous command languishes in the low priority queue.

I'd guess the root of it is that Publishing API should do nested commands with same priority as the initial one, but since this is all based on update_type it's quite hard to unpick.

Reply:
Thanks Kevin - I'll rewrite the PR, as your summary is more accurate 👍

Reply:
Done - please take a look :)

Reply:
:+1: